packages/jobs/lib/runner/render.runner.ts

```diff
@@ -1,15 +1,50 @@
 import type { Runner } from './runner.js';
 import { RunnerType } from './runner.js';
 import { ProxyAppRouter, getRunnerClient } from '@nangohq/nango-runner';
-import { getEnv, getPersistAPIUrl } from '@nangohq/shared';
-import api from 'api';
+import { AxiosResponse, NodeEnv, getEnv, getPersistAPIUrl } from '@nangohq/shared';
 import tracer from 'dd-trace';
+import axios, { AxiosInstance } from 'axios';
 
-const render = api('@render-api/v1.0#aiie8wizhlp1is9bu');
-render.auth(process.env['RENDER_API_KEY']);
+export class RenderAPI {
```

Comment:
nitpicking, class could be in a separate file

Reply:
sure